<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comfortaa&quot;;
font: bold;
color:  rgb(56, 84, 102);</string>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comfortaa&quot;;
font: bold;
color:  rgb(56, 84, 102);</string>
   </property>
   <property name="text">
    <string>Логин </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Comfortaa&quot;;
font: bold;
color:  rgb(56, 84, 102);</string>
   </property>
   <property name="text">
    <string>Войдите в систему</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comfortaa&quot;;
font: bold;
color:  rgb(56, 84, 102);
border:3px solid #DDEBFF;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comfortaa&quot;;
font: bold;
color:  rgb(256, 84, 102);
border:3px solid #DDEBFF;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comfortaa&quot;;
font: bold;
color:  rgb(256, 84, 102);
border:3px solid #DDEBFF;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Comfortaa&quot;;
font: bold;
color:  rgb(56, 84, 102);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_showPassword">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imasdasd/openEye.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comfortaa&quot;;
font: bold;
color:  rgb(56, 84, 102);
border:3px solid #DDEBFF;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Регестрация</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
